starlark/int_generic.go

```diff
@@ -0,0 +1,33 @@
+//+build !linux,!darwin
```

Comment:
This pair of lines should be one line:

// +build !linux,!darwin !amd64,!arm64,!mips64x,!ppc64x

Reply:
Good catch. I always forget whether space means AND or OR.